contributionguidelines.md

```diff
@@ -0,0 +1,21 @@
+# How to Contribute
+> Submit a PR (Pull Request)
+
+Feel free to contribute any learning resource pertaining to any subject that you would like to share with the Dev Community.
+
+- Make A Branch
+  * Please create a separate branch for each contribution that you're working on. Do not make changes to the default branch (e.g. main, develop) of your fork.
```

Comment:
It's probably a good idea to call out the need to fork the project and push your branch to your fork and not attempt to push to this repository directly.

Reply:
Great Idea! will add.